controllers/postgresqluser_controller.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package controllers
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	azurev1alpha1 "github.com/Azure/azure-service-operator/api/v1alpha1"
+)
+
+// PSqlServerPort is the default server port for psql server
+const PSqlServerPort = 5432
+
+// DriverName is driver name for db connection
+const PDriverName = "postgres"
+
+// SecretUsernameKey is the username key in secret
+const PSecretUsernameKey = "username"
+
+// SecretPasswordKey is the password key in secret
+const PSecretPasswordKey = "password"
+
+// PSQLUserFinalizerName is the name of the finalizer
+const PSQLUserFinalizerName = "psqluser.finalizers.azure.com"
+
+// PostgreSQLUserReconciler reconciles a PSQLUser object
+type PostgreSQLUserReconciler struct {
+	Reconciler *AsyncReconciler
+}
+
+// +kubebuilder:rbac:groups=azure.microsoft.com,resources=PostgreSQLUsers,verbs=get;list;watch;create;update;patch;delete
```

Comment:
I think this should be all lowercase. Not sure if it will affect anything if it is not, but when kube builder generates this for us, it is lowercase